buildenv/ansible/roles/openj9-build-environment/tasks/main.yml

```diff
@@ -19,40 +19,36 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 ###############################################################################
+# (c) Copyright IBM Corp. 2020, 2020 All Rights Reserved
```

Comment:
Pls use the standard ` Copyright (c) 2020, 2020 IBM Corp. and others` copyright, used by most of the OpenJ9 files.

Reply:
I copied that text from some other file (in jdk8?). Updated as requested.

Reply:
Yes, the ibmruntimes repos use different copyrights.